OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -1,6 +1,24 @@
 # Oracle WebCenter Portal 12.2.1.4.0 on Docker
+
+## Mount a host directory as a data volume
+ The default location of the volume in container is under /var/lib/docker/volumes. There is an option to mount a directory from the host into a container as volume. In this project we will use that option for the data volume.
+To mount a host directory `/scratch/wcpdocker/volumes/wcpportal`(`DATA_VOLUME`) as a data volume, execute the below command.
+
+> The userid can be anything but it must belong to uid:guid as 1000:1000, which is same as 'oracle' user running in the container.
```

Comment:
"userid" should be "User ID" but it can't be anything if you're then telling us it must be `1000:1000`. 

Please use backtick quotes around `uid`, `gid`, `1000:1000` and `oracle`. You're also missing the word "the" before `oracle`.